packs/device42/actions/suggest_next_ip.py

```diff
@@ -0,0 +1,36 @@
+import requests
+from st2actions.runners.pythonrunner import Action
+
+
+class SuggestNextIp(Action):
+    def run(self, subnet_id=None, subnet=None, name=None, vrf_group_id=None, vrf_group=None,
+            reserved_ip=None):
+        d42_server = self.config.get('d42_server', None)
+        if not d42_server:
+            raise ValueError('"d42_server" config value is required')
+
+        d42_username = self.config.get('d42_username', None)
+        if not d42_username:
+            raise ValueError('"d42_username" config value is required')
+
+        d42_password = self.config.get('d42_password', None)
+        if not d42_password:
+            raise ValueError('"d42_password" config value is required')
+
+        verify = False
+        if self.config.get('verify_certificate', None) == 'true':
```

Comment:
Those lines seem to be duplicated across all the actions.

To reduce the duplication you can move those checks into a utility method on a new base class (e.g. `BaseDevice42Action`) from which all other action classes inherit from. Then you can call that utility method inside the actions.